docs/router/open-telemetry/custom-attributes.mdx

```diff
@@ -0,0 +1,144 @@
+---
```

Comment:
This was moved from `global-custom-attributes.mdx`, but I think since there are too many changes, git doesn't detect it as a move.